content/it/partners/_index.html

```diff
@@ -9,10 +9,101 @@
 <section id="users">
     <main>
         <h5>Kubernetes collabora con i partner per creare una base di codice forte e vivace che supporti uno spettro di piattaforme complementari.</h5>
```

Comment:
This is the literal translation of the English sentence but honestly, it does not sound great in Italian. I personally do not understand what this means (grammatically is correct though). I think we should rephrase this to something else.

Reply:
Hi @mattiaperi, any idea how we can rephrase this sentence?
Thanks in advance for your help